kiwi/repository/yum.py

```diff
@@ -248,9 +258,9 @@ def _create_runtime_config_parser(self):
         self.runtime_yum_config.set(
             'main', 'gpgcheck', self.gpg_check
         )
-        self.runtime_yum_config.set(
-            'main', 'repo_gpgcheck', self.gpg_check
-        )
+        # We are not setting repo_gpgcheck, which forces repository
+        # signature checks, because I could not manage to make it work.
+        # Nevertheless, package signature validation is functional.
```

Comment:
Can you try checking on CentOS 7 and Fedora 25 systems? If the behavior is broken/fixed in yum, I'd like to know when it was. That might help in identifying when it got fixed (if it did at all) and possibly backport it to the SUSE yum package.